tests/test_sd_whonix.py

```diff
@@ -2,9 +2,22 @@
 import json
 from jinja2 import Template
 
+
 from base import SD_VM_Local_Test
 
 
+def skip_if_v3_is_missing():
```

Comment:
nit: this method name in confusing since now we support v2 or v3 within the same config option